### ISSUE TITLE
Add user management

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#17]
Some informations and actions should be only accessible to an administrator.
Some informations and actions should be only accessible to its owner(s) and administrator(s).
The administrator should be the one and the only one to be able to manage this user management (create user, delete user, attach/detach to project etc...).

Technically, I think it means adding some tables in DB + handle sessions etc.. in web UI.

**Comment by melezhik** [comment]
yeah, this is big and probably very valuable feature request. I am still thinking where this should be part of Sparky of maybe other frontend ( File Triggering Protocol ... )

**Comment by melezhik** [comment]
we can consider HTTP BASIC AUTH support as a good start for that - https://github.com/melezhik/sparky#http-basic-authentication-support
